index.js

```diff
@@ -48,6 +51,66 @@ class ExecutorQueue extends Executor {
         this.fuseBox = new FuseBox();
         this.fuseBox.addFuse(this.queueBreaker);
         this.fuseBox.addFuse(this.redisBreaker);
+
+        this.scheduler = new Resque.Scheduler({ connection: redisConnection });
+    }
+
+    /**
+     * Starts new periodic builds in an executor
+     * @method _periodicStart
+     * @param {Object} config               Configuration
+     * @param {Object} config.pipeline      Pipeline of the job
+     * @param {Object} config.job           Job object to create periodic builds for
+     * @param {Object} config.tokenGen      Function to generate JWT from username, scope and scmContext
+     * @return {Promise}
+     */
+    async _periodicStart(config) {
+        const pipeline = config.pipeline;
+        const job = config.job;
+        const tokenGen = config.tokenGen;
+
+        await this.scheduler.connect();
```

Comment:
Do not create new connection everytime, it creates some weird effect. Try to do something like `this.connect()` function: https://github.com/screwdriver-cd/executor-queue/blob/608e644846536afb012f1eaba0acf8f71725c393/index.js#L171